src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {'age' in person && <p className="Person__age">I am {age}</p>}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName} is my {partner}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
Now a tag with the same class is duplicated twice, you need to remove this duplication and enter the check inside the tag